commit 7f74206b39673b1a110285c6992f70507e211060
Author: Tim Parenti <tim@timtimeonline.com>
Date:   Thu Jul 22 10:17:52 2021 -0400

    No leap second on 2021-12-31
    
    Per IERS Bulletin C 62 (2021-07-05).
    https://hpiers.obspm.fr/iers/bul/bulc/bulletinc.62
    
    * leap-seconds.list: Update file from NIST, retrieved from
    ftp://ftp.nist.gov/pub/time/leap-seconds.list

diff --git a/leap-seconds.list b/leap-seconds.list
--- a/leap-seconds.list
+++ b/leap-seconds.list
@@ -204,10 +204,10 @@
 #	current -- the update time stamp, the data and the name of the file
 #	will not change.
 #
-#	Updated through IERS Bulletin C61
-#	File expires on:  28 December 2021
+#	Updated through IERS Bulletin C62
+#	File expires on:  28 June 2022
 #
-#@	3849638400
+#@	3865363200
 #
 2272060800	10	# 1 Jan 1972
 2287785600	11	# 1 Jul 1972
@@ -252,4 +252,4 @@
 #	the hash line is also ignored in the
 #	computation.
 #
-#h 	2ab8253d d4380d28 75f01343 381504f8 8f8a4bfc
+#h 	599d45bf accd4b4f 8b60e46 49b623 7d13b825
